unit_test/docker_container_test.py

```diff
@@ -94,12 +94,12 @@ def test_binary_output():
 
         data = bytes(output, encoding='utf8', errors='surrogateescape')
 
-        for i in range(0, 512):
+        for i in range(512):
             assert data[i] == i % 256
 
         # check that environment variables can carry binary data, except null characters
         # (https://www.gnu.org/software/libc/manual/html_node/Environment-Variables.html)
-        binary_data = bytes(n for n in range(1, 256))
+        binary_data = bytes(range(1, 256))
```

Comment:
I'm less sure about this change, myself. Clearly they're equivalent, but somehow `binary_data = bytes(n for n in range(1, 256))` is more obviously a buffer containing every byte value from 1 to 256 than `bytes(range(1, 256))`, which looks like it was written by a computer :P
```suggestion
        binary_data = bytes(n for n in range(1, 256))
```

Reply:
I like the simpler one that does not make an extra generator, but okay. Given the comment above especially, I don't `bytes(range(` could be misunderstood.